sass/responsive/_desktop.scss

```diff
@@ -1,5 +1,34 @@
 @charset 'UTF-8';
 
+@media screen and (min-width: 1680px) {
+    .sidebar--right {
+        width: 500px;
+        @include translateX(100%);
+
+        &.move--left {
+            @include translateX(0);
+        }
+    }
+
+    .inner-wrap {
+        &.move--left {
+            .search__form {
```

Comment:
Consider avoiding nesting and using classes instead.

Reply:
@nevyangelova I'm sorry, I don't understand. Isn't nesting already using classes?
Can you provide an example of what you mean?

Reply:
I tried simplifying it, but the reason why we have `move--left` is because that class is added dynamically, and we want to apply certain styles only if that class is there.

And the nesting for `app__content` is there because there are 2 `post-create` containers within the app, and I only want to target the one directly within `app__content`.